templates/all-services/reports/a11y-1.json

```diff
@@ -0,0 +1,648 @@
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The success criterion was tested with WAVE. The WAVE browser extensions allows to evaluate web content for accessibility issues directly within a browser.",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:audio-control",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:contrast-minimum",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The contrast ratio was tested with WCAG Contrast checker a FrieFox add-on",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:resize-text",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The text was increased to 200% in Firefox browser to ensure text can be enlarged.",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:image-of-text",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:reflow",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The success criterion was tested in Firefox Responsive Design Mode to ensure content can be enlarged without increasing line length.",
+      "earl:outcome": "earl:passed",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:non-text-contrast",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The contrast ratio was tested with WCAG Contrast checker a FrieFox add-on to ensure visual information meets the same minimum contrast",
```

Comment:
Fix typo, "FrieFox" changed to "Firefox".

Reply:
Fixed